src/components/BookPage.js

```diff
@@ -0,0 +1,22 @@
+import { v4 as uuidv4 } from 'uuid';
+import Book from './book';
+import Form from './form';
+
+const BookPage = () => {
+  const books = [
+    { id: uuidv4(), title: 'The Great Gatsby', author: 'Scott Fitzgerald' },
+    { id: uuidv4(), title: 'Moby Dick', author: 'Herman Melville' },
+    { id: uuidv4(), title: 'War and Peace', author: 'Leo Tolstoy' },
+    { id: uuidv4(), title: 'The Brothers Karamazov', author: 'Fyodor Dostoyevsky' },
+  ];
+
+  return (
+    <>
+      {/* rander */}
+      <Book key={books.id} title={books.title} author={books.author} BookList books={books} />
```

Comment:
:red_circle: [REQUIRED]

1. You've correctly defined the BookList component to accept a books prop, but in the BookPage component, you should pass the books array to BookList like this:

`<BookList books={books} />`
This way, the BookList component can access the list of books you've defined.